### PR TITLE
Update Piper add-on to 1.3.2

### PR DESCRIPTION
**Pull request by synesthesiam**
- Add voices for Hungarian, Turkish, Portuguese, Slovak, and Czech
- Add `upgrade_voices` and `debug_logging` options
- Upgrade to Debian bookworm

### CODE REVIEW

File: piper/CHANGELOG.md
Comment (balloob) [inline piper/CHANGELOG.md:9]: This was 1.1.0 before?

Comment (synesthesiam) [reply piper/CHANGELOG.md:9]: Yeah, I made a mistake last time.

File: piper/translations/en.yaml
Comment (frenck) [inline piper/translations/en.yaml:41]:
```suggestion
      Download the list of new voices automatically every time the add-on starts.
      You must also reload the Wyoming integration for Piper in Home Assistant
      to see new voices.
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: piper/Dockerfile
Comment (synesthesiam) [reply piper/Dockerfile:20]: The Python in Debian bookworm gives this error when trying to upgrade system packages:

```
$ pip3 install --upgrade pip
error: externally-managed-environment

× This environment is externally managed
╰─> To install Python packages system-wide, try apt install
    python3-xyz, where xyz is the package you are trying to
    install.
    
    If you wish to install a non-Debian-packaged Python package,
    create a virtual environment using python3 -m venv path/to/venv.
    Then use path/to/venv/bin/python and path/to/venv/bin/pip. Make
    sure you have python3-full installed.
    
    If you wish to install a non-Debian packaged Python application,
    it may be easiest to use pipx install xyz, which will manage a
    virtual environment for you. Make sure you have pipx installed.
    
    See /usr/share/doc/python3.11/README.venv for more information.
```

There is likely a flag to override, but I figured it was best to go with the flow.

Comment (frenck) [reply piper/Dockerfile:20]: I don't agree, as this is not a normal user runtime.

I'd suggest restoring the original and set the `PIP_BREAK_SYSTEM_PACKAGES=1` on the environment.

File: piper/config.yaml
Comment (balloob) [inline piper/config.yaml:33]: Any reason we make this an option and not always on ?

File: piper/Dockerfile
Comment (synesthesiam) [reply piper/Dockerfile]: Removed the venv

File: piper/config.yaml
Comment (synesthesiam) [reply piper/config.yaml:33]: It unfortunately prints a line each time the healthcheck hits (which asks for available voices). So you'd end up with quite a few log messages over time with this on.

**Review by frenck — APPROVED** [review]
✅ Approve from my end, not sure if the above open discussions need to be resolved before merging this one in.

../Frenck

File: piper/config.yaml
Comment (agners) [inline piper/config.yaml:23]:
```suggestion
  update_voices: true
```

File: piper/config.yaml
Comment (agners) [reply piper/config.yaml:33]: @synesthesiam  I think @balloob was referring to the lower option (`update_voices`). I agree it might be a good idea to do this by default on startup, but maybe there are reasons we don't want it to be enabled by default?

File: piper/config.yaml
Comment (synesthesiam) [reply piper/config.yaml:23]: OK, let me make the loading a bit more robust and I'll enable it by default.

**Comment by synesthesiam** [comment]
@agners Ready to merge.

File: piper/CHANGELOG.md
Comment (agners) [inline piper/CHANGELOG.md:3]:
```suggestion
## 1.3.2
```

**Review by agners — APPROVED** [review]

**Review by frenck — APPROVED** [review]

**Pull request merged**
